Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -252,21 +273,35 @@ class FirefoxHomeViewController: UICollectionViewController, HomePanel {
     func scrollToTop(animated: Bool = false) {
         collectionView?.setContentOffset(.zero, animated: animated)
     }
+    
+    func configureBookmarksForRecentlySaved() {
+        profile.places.getRecentBookmarks(limit: 5).uponQueue(.main) { [weak self] result in
+            if let bookmarks = result.successValue,
+               !bookmarks.isEmpty,
+               !BookmarksHelper.filterOldBookmarks(bookmarks: bookmarks, since: 10).isEmpty {
+                self?.hasRecentBookmarks = true
+            } else {
+                self?.hasRecentBookmarks = false
+            }
+        }
+    }
+    
```

Comment:
@nishant2718: Just a reminder comment to update this method so that our collectionview reloads once we have proper state of bookmark